### ISSUE TITLE
Requirements / Features

### ISSUE THREAD

**Issue by kmpaul** [NCAR/xdevbot#1]
- [x] **Feature:** Configuration of the bot is given (essentially) by a mapping of boards (1 for each campaign) to "watched" repos
```python
watchlists = {
    'core': set(['NCAR/xdev', 'NCAR/xdevbot', ...]),
    'eocb': set(['NCAR/python-tutorial', ...]), 
    'search': set([]),
    'ingestion': set(['intake/intake-esm', ...]),
    'analysis': set(['xarray-contrib/cf-xarray', ...]),
    'checkpoint': set(['NCAR/xpersist', ...]),
    'publication': set([]),
    'platform': set(['NCAR/jupyter-forward', ...])
```

- [x] **Feature:** A main/aggregate board should exist that "watches" all repos from all campaign boards
```python
main_watchlist = set.union(*watchlists.values())
```
- [x] **Feature:** Each board should have the following columns: `New`, `Easy`, `Low Priority`, `High Priority`, `In Progress`, `Stalled`, `For Review`, `Done`

- [x] **Feature:** New or reopened issues/PRs on watched repos get added to the `New` column on the appropriate board

- [x] **Feature:** Issues/PRs that are closed/merged are moved to the "Done" column

- [x] **Feature:** Users can add individual issues/PRs to a board manually

- [x] **Feature:** Adding a repo to a board's "watch list" should be a single-step process
  - e.g., add the `xdev-bot` user to the repo to be watched, and the bot polls every X seconds for open invitations and accepts them

- [x] **Feature:** Manually added issues/PRs are controlled by polling functions

**Comment by kmpaul** [comment]
@andersy005 There are 2 issue cards in the New column of the Xdev Core project board that the polling functions *should* manage. But they aren’t being controlled. Not sure why.

**Comment by andersy005** [comment]
> New column of the Xdev Core project board that the polling functions should manage. But they aren’t being controlled. Not sure why.


I fixed the polling issue in #20

**Comment by kmpaul** [comment]
I think polling is working in a "draft" state.  Let's submit new issues as we find them.  I'll close this now.
